index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign up page project</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <header>
+        <div class="infoBar">
+            <div>
+                <p>Already signed up?</p>
+                <a href="www.facebook.com"><p>Do you forget your password?</p></a>
+            </div>
+            <div class="inputbutton">
+                <input type="button" name="signin" value="Sign in"/>
+            </div>   
+        </div>
+    </header>
+    <section class="Main-header">
+        <div>
+            <h1>intuit</h1>
+            <div class="Header-logos">
+                <img src="./assets/mintLogo.png" alt="Mint" />
+                <h4>mint</h4>
+                <img src="./assets/qukBooksIcon.png" />
+                <h4>quickbooks</h4>
+                <img src="./assets/Turbotax.png" />
+                <h4>turbotax</h4>
+            </div>
+        </div>
+    </section>
+    <main>
+        <section>
+            <form>
+                <fieldset>
+                    <div class="form-header">
+                        <h2>Create an Intuit account</h2>
+                        <p>One account for everything Intuit, including Mint.</p>
+                        <a href="https://www.w3schools.com">Learn More</a>
+                    </div>
+                    <div class="form-content">
+                        <label>
+                            <div>
+                                E-mail
+                            </div> 
+                            <input type="email" name="email" /> 
+                        </label>
+                        <label>
+                            <div>
+                                Phone
+                            </div>
+                            <input type="text" name="number" /> 
+                        </label>
+                        <label>
+                            <div>
+                                Password
+                            </div>
+                            <input type="password" name="password" /> 
+                        </label> 
+                    </div>
+                    <div class="form-input-button">
+                        <input type="button" value="Sign in" />
```

Comment:
The form should have a button with type submit:
- [ ] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))